packages/atlas/src/providers/auth/auth.provider.tsx

```diff
@@ -131,27 +134,35 @@ export const AuthProvider: FC<PropsWithChildren> = ({ children }) => {
         action: 'login',
       }
       let signatureOverPayload = null
-      let plainSeed: string | null = null
+      let localSeed: string | null = null
       if (params.type === 'internal') {
         const { email, password } = params
         const id = (await scryptHash(`${email}:${password}`, '0x0818ee04c541716831bdd0f598fa4bbb')).toString('hex')
         const data = await getArtifacts(id, email, password)
         if (!data) {
-          setInitializationState('needAuthentication')
+          setIsAuthenticating(false)
           return {
             data: null,
             error: LogInErrors.ArtifactsNotFound,
           }
         }
         const { keypair, decryptedSeed } = data
-        plainSeed = decryptedSeed
+        localSeed = decryptedSeed
         payload.joystreamAccountId = keypair.address
         signatureOverPayload = u8aToHex(keypair.sign(JSON.stringify(payload)))
       }
 
       if (params.type === 'external') {
         payload.joystreamAccountId = params.address
-        signatureOverPayload = await params.sign(JSON.stringify(payload))
+        try {
+          signatureOverPayload = await params.sign(JSON.stringify(payload))
+        } catch (_) {
```

Comment:
```suggestion
        } catch {
```
As far as I remember you can omit parentheses.